web/rootfs/defaults/meet.conf

```diff
@@ -134,6 +134,20 @@ location ^~ /etherpad/ {
 }
 {{ end }}
 
+{{ if $WHITEBOARD_ENABLED }}
+# whiteboard (excalidraw-backend)
+location = /socket.io/ {
```

Comment:
Let's make this /excalidraw/socket-io

Reply:
That is because the URL in the setting needs to end in /excalidraw/

Reply:
Which settings exactly? I changed both `meet.conf` and `settings-config.js`, the latter being the file i thought was responsible for the Excalidraw javascript frontend making HTTP requests?

Reply:
In both.

Reply:
I did: it doesn't work. Please read my full comment here https://github.com/jitsi/docker-jitsi-meet/pull/1480#issuecomment-2016579181

> However as soon as i move the excalidraw location to a different path as requested in the code review things start to break:
```diff
diff --git a/web/rootfs/defaults/meet.conf b/web/rootfs/defaults/meet.conf
index 986e9da..6ade033 100644
--- a/web/rootfs/defaults/meet.conf
+++ b/web/rootfs/defaults/meet.conf
@@ -136,7 +136,7 @@ location ^~ /etherpad/ {
 
 {{ if $WHITEBOARD_ENABLED }}
 # whiteboard (excalidraw-backend)
-location = /socket.io/ {
+location = /excalidraw-collab/socket.io/ {
     tcp_nodelay on;
 
     proxy_http_version 1.1;
diff --git a/web/rootfs/defaults/settings-config.js b/web/rootfs/defaults/settings-config.js
index 9fae33a..a828a2d 100644
--- a/web/rootfs/defaults/settings-config.js
+++ b/web/rootfs/defaults/settings-config.js
@@ -557,7 +557,7 @@ config.whiteboard = {
 {{ if .Env.WHITEBOARD_COLLAB_SERVER_PUBLIC_URL -}}
     config.whiteboard.collabServerBaseUrl = '{{ $WHITEBOARD_COLLAB_SERVER_PUBLIC_URL }}';
 {{ else }}
-    collabServerBaseUrl: '{{ $PUBLIC_URL }}'
+    collabServerBaseUrl: '{{ $PUBLIC_URL }}/excalidraw-collab/'
 {{ end }}
 };

```
> For some reason my browser keeps trying to connect to $PUBLIC_URL/socket.io instead of $PUBLIC_URL/excalidraw-collab/socket.io.

Reply:
Ok, we can look into that later then.